worker/src/Consumer/startListener.ts

```diff
@@ -0,0 +1,32 @@
+import { WorkHandler, WorkOptions } from "pg-boss";
+import { drainQueue } from "./migrate";
+import { getConsumer } from "./getConsumer";
+import pino from "pino";
+import config from "config";
+const logger = pino();
+
+export async function startListener<T>(queueName: string, handler: WorkHandler<T>, options?: WorkOptions) {
```

Comment:
Could we default options to an empty object to avoid the if statement at the bottom of the function?